projects/ltc2378_fmc/zed/README.md

```diff
@@ -0,0 +1,8 @@
+# LTC2378-FMC/ZED HDL Project
```

Comment:
Add the directives and VADJ value which was tested in hardware. Example: https://github.com/analogdevicesinc/hdl/blob/main/projects/ad353xr/coraz7s/README.md?plain=1#L1-L5